### DISCUSSION TITLE
Github Actions disabled

### DISCUSSION THREAD

**Discussion by shubhankartrivedi** [community/community#53016] [Actions]
### Select Topic Area

Question

### Body

It has been a year and no one is replying my support ticket.... 
Im just done with GitHub....

What I do....
I have opened a new ticket but still no is replying for a month...
Sent messages there 3 times...

Github Support dead or what...

**Comment by kvten** [comment]
Settings to toggle in repo:
- Allow all actions and reusable workflows
- Read and write permissions

Make sure actions yml file is in `.github/workflows` folder

**Reply by shubhankartrivedi** [reply]
It said "Github Actions disabled on my account".

**Reply by BartVanGestel** [reply]
Github actions is a part of an organization or a repository and should be enabled there.
Go to:
- your organization/repository:settings
- actions
- general
- `allow all actionsand reusable workflows`

**Comment by shubhankartrivedi** [comment]
<img width="827" alt="Screenshot 2023-04-18 at 3 29 39 PM" src="https://user-images.githubusercontent.com/78972053/232743001-8f2e6814-c994-4e7c-95b3-4c8186be85a9.png">


I have already did it. it's not working...

**Comment by MakePixelsWork** [comment]
We have the same problem.
